go.mod

```diff
@@ -5,7 +5,8 @@ go 1.23.7
 toolchain go1.24.2
 
 require (
-	github.com/ClickHouse/ch-go v0.65.1
+	github.com/ClickHouse/ch-go v0.66.0
```

Comment:
is this a first-class dependency? the code seems to only depend on `clickhouse-go`

Reply:
@yurishkuro This dependency is being used elsewhere right now but we're planning on removing it and using only clickhouse-go. It will be removed in a follow-up PR.